lib/handlers/shared.ts

```diff
@@ -162,6 +162,27 @@ export const INTENT_SPECIFIC_CONFIG: { [key: string]: IntentSpecificConfig } = {
   },
 }
 
+export type FeeOnTransferSpecificConfig = {
+  enableFeeOnTransferFeeFetching?: boolean
+  useCachedRoutes?: boolean
+}
+
+// TODO: ROUTE-86 - remove useCachedRoutes: !enableFeeOnTransferFeeFetching once we are ready to consume from RoutesDB
```

Comment:
perfect, so this is just for testing right? just confirming

Reply:
Yep this is only used for testing. integ-test has added the [assertion](https://github.com/Uniswap/routing-api/pull/362/files#diff-e193a17cd7d38af77f287fe6bcf1c109b80c24bff4f9a4e317e4c95ea6a1c047R1054) that only in case of `enableFeeOnTransferFeeFetching` = true, we will not hit the cached routes.